package.json

```diff
@@ -91,8 +91,7 @@
   },
   "peerDependencies": {
     "apollo-client": "^2.3.8",
-    "react": "^15.0.0 || ^16.0.0",
-    "react-dom": "^15.0.0 || ^16.0.0",
+    "react": "0.14.x || 15.* || ^15.0.0 || ^16.0.0",
```

Comment:
Careful, I think you might be branching off an older version of `master`, since this seems to be undoing recent changes.

Reply:
Oh, will check that. Sometimes it's pretty weird keeping up with an upstream sorry bout that